Change the order querying bybit (#9388)
rotki/rotki: docs/changelog.rst

* :bug:`-` Some specific cases of yearn v2 vault deposit/withdrawals that had problems will now be decoded properly.
* :bug:`-` Deleting an ethereum address will now remove the withdrawals cache for that address so re-adding it will now properly detect ethereum staking withdrawals again.
* :bug:`-` Fix double count of cowswap fees.

> * :bug:`-` rotki won't exceed now the two years limit when requesting trades from Bybit.

Reviewer: ```suggestion
* :bug:`-` rotki will no longer exceed the two years limit when requesting trades from Bybit.
```
